web/models/stripe_account.ex

```diff
@@ -0,0 +1,38 @@
+defmodule CodeCorps.StripeAccount do
+  @moduledoc """
+  Represents a StripeAccount stored on Code Corps.
+  """
+
+  use CodeCorps.Web, :model
+
+  @primary_key {:id, :string, []} # set primary key as string
+  @derive {Phoenix.Param, key: :id} # set method of determining primary key for route
```

Comment:
Whoa, interesting, I haven't heard of `derive` until just now. From what I just briefly read about the use of `derive`, it seems `derive` is a great way to [set custom primary keys](http://www.phoenixframework.org/docs/ecto-custom-primary-keys). Since we're using `id` as the primary key, I'm not certain we want to use `derive` here. Same thing for `@primary_key`.